cli/azd/internal/repository/app_init.go

```diff
@@ -244,30 +244,31 @@ func (i *Initializer) InitFromApp(
 
 	// Create the infra spec
 	var infraSpec *scaffold.InfraSpec
-	if !i.features.IsEnabled(alpha.Compose) { // backwards compatibility
+	composeEnabled := i.features.IsEnabled(alpha.Compose)
```

Comment:
Can you move the `alpha.Compose` FeatureId to the package that exposes the feature?
The alpha package should not be aware of what alpha features exists in the product, it is a package that provides the functionality to define and use the features, but features should be defined where they belong.

Reply:
Just to clarify, are we changing directions from our previous efforts to add alpha feature string constants into `alpha_features.go`, as in #1715, where we added [terraform](https://github.com/azure/azure-dev/blob/main/cli/azd/pkg/alpha/alpha_features.go), which it still remains to date?

Keep in mind that this is indeed a string constant, it is not any meaning functionality that would violate any directional dependencies.

Reply:
the terraform constant is dead code (not used anywhere). We should just remove it. 
alpha feature constants should not be defined within the alpha package.